app/templates/fx_ios/release/listing_sept_2015.php

```diff
@@ -115,10 +115,23 @@
     }
 }
 
-$whatsnew = function ($translations) use ($_) {
+if (in_array($current_locale, ['de', 'en-US'])) {
+    $whatsnew = function ($translations) use ($_) {
         return <<<OUT
-• {$_('Tracking protection is now enabled in private browsing mode by default. Firefox will automatically block third party trackers, which limits ads following you around the internet and increases browsing speed.')}
-• {$_('Improved sync functionality between your mobile and desktop versions of Firefox. Update passwords, history and bookmarks on mobile and access them on your desktop browser (and vice versa).')}
-• {$_('Compatibility with iOS 11.')}
+• {$_('Firefox has a new look and feel that we call “Photon”. It’s a modern design that puts users’ needs first - with easier access to the menu, rearranged menu items and an updated minimalist look. Take a peek!')}
```

Comment:
Looks like you changed from `“Photon.”` (in the request) to `“Photon”.`.

I think English includes punctuation in quotes? If you need to change this, you'll also need to change the next one, and update the files in appstores.